src/main/java/no/nav/foreldrepenger/stønadskonto/regelmodell/regler/LeggTilDagerVedFlereBarn.java

```diff
@@ -21,7 +22,7 @@ public LeggTilDagerVedFlereBarn() {
     @Override
     public Evaluation evaluate(KontoerMellomregning mellomregning) {
         var grunnlag = mellomregning.getGrunnlag();
-        var parametertypeFlerbarn = mellomregning.getGrunnlag().getAntallBarn() == 2 ? Parametertype.EKSTRA_DAGER_TO_BARN : Parametertype.EKSTRA_DAGER_TRE_ELLER_FLERE_BARN;
+        var parametertypeFlerbarn = flerbarnsParametertype(grunnlag).orElseThrow();
         mellomregning.getKontokonfigurasjon().add(new Kontokonfigurasjon(StønadskontoKontotype.TILLEGG_FLERBARN, parametertypeFlerbarn));
         if (aktivitetsKravIkkeMinsterett(grunnlag)) {
             mellomregning.getKontokonfigurasjon().add(new Kontokonfigurasjon(StønadskontoKontotype.FLERBARNSDAGER, parametertypeFlerbarn));
```

Comment:
Synes TILLEGG_FLERBARN og FLERBARNSDAGER er litt defuse.

FLERBARNSDAGER settes alltid for begge, og for BFHR før WLB?

Reply:
Er det feil å tenke at UFØREDAGER og FLERBARNSDAGER kan begge utledes i LeggTilDagerVedBareFarRett? 

Jeg vet ikke helt hva funksjonen til denne FLERBARNSDAGER typen ved BEGGE_RETT. Tanker?

Reply:
FLERBARNSDAGER går på aktivitetskrav (begge rett og BFHR før WLB). BFHR post WLB har dem som minsterett.
TILLEGG_FLERBARN er dagene som legges på fellesperiode/foreldrepenger uansett.De er skilt ut ifm stønadsstatistikk,
Kan evt endre navn på de to aktivitetskrav-relaterte parametrene. Ser ikke noen fundamentale forenklinger.

Reply:
Konseptet "fritak for aktivitetskrav" har blitt slått sammen med minsterett ifm WLB og semantikken for trekk av dager ble ganske annerledes med WLB

Reply:
Hovedalternativet er å eksponere ut alle parametre og la fpsak regne ut minsterett ut fra minsterett, uføre og flerbarn på vei inn i uttaksreglene

Reply:
Skjønner. Takk for avklaring 👍